x-pack/legacy/plugins/graph/public/components/graph_visualization/_graph_visualization.scss

```diff
@@ -0,0 +1,60 @@
+.gphEdge {
+  fill: $euiColorMediumShade;
+  stroke: $euiColorMediumShade;
+  stroke-width: 2;
+  stroke-opacity: 0.5;
+
+  &:hover {
+    stroke-opacity: 0.95;
+    cursor: pointer;
+  }
+
+  &--selected {
+    stroke: $euiColorDarkShade;
+    stroke-opacity: 0.95;
+  }
+
+  &--inferred {
+    stroke-dasharray: 5, 5;
+  }
+}
+
+.gphNode__label {
+  @include gphSvgText;
+  cursor: pointer;
+  &--html {
+    @include euiTextTruncate;
+    text-align: center;
+  }
+}
+
+.gphNode__markerCircle {
+  fill: $euiColorDarkShade;
+  stroke: $euiColorEmptyShade;
+}
+
+.gphNode__markerText {
+  @include gphSvgText;
+  font-size: $euiSizeS - 2px;
+  fill: $euiColorEmptyShade;
+}
+
+.gphNode__circle {
+  fill: $euiColorMediumShade;
+  &--selected {
+    stroke-width: $euiSizeXS;
+    stroke: transparentize($euiColorPrimary, 0.25);
+  }
+}
+
+.gphNode__text {
+  fill: $euiColorDarkestShade;
+
+  &--lowOpacity {
```

Comment:
Remove this class completely. The icons should be fully opaque when on top of the node colors.

Reply:
Good point. This class was still used in other places because the styling of icons in menus was entangled with the node styles. I separated them and removed some unused classes.